app/javascript/src/hyacinth_v1/components/controlled_vocabularies/terms/TermForm.js

```diff
@@ -123,11 +123,10 @@ function TermForm(props) {
           (() => {
             if (formType === 'edit') {
               return <PlainText value={uri} />;
-            } else if (termType === 'external' || termType === '') {
-              return <TextInput sm={inputColWidth} value={uri} onChange={v => setUri(v)} />;
-            } else {
-              return <ReadOnlyInput sm={inputColWidth} value={uri} />;
+            } if (termType === 'external' || termType === '') {
```

Comment:
do you want a newline here?

Reply:
Whoa, weird auto-reformat for my IDE to suggest/perform.  Yes I do!  Good catch, thanks!